src/screens/MemberDetail/UserContactDetails.tsx

```
@@ -0,0 +1,596 @@
+                        alt={tCommon('userImage')}
+                        size={60}
+                        dataTestId="profile-picture"
+                        radius={150}
+                      />
+                    )}
+                    <i
+                      className={`fas fa-edit position-absolute bottom-0 right-0 p-2 bg-white rounded-circle ${styles.userContactDetailContactAvatarEditIcon}`}
+                      onClick={() => fileInputRef.current?.click()}
+                      data-testid="uploadImageBtn"
+                      title={tCommon('userEditProfilePicture')}
+                      role="button"
```

Biome (2.1.2)

[error] The HTML element i is non-interactive and should not have an interactive role.

Replace i with a div or a span.
Unsafe fix: Remove the role attribute.


(lint/a11y/noNoninteractiveElementToInteractiveRole)